pkg/namespace/namespace_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2019 The Tekton Authors.
```

Comment:
Same as above `// Copyright © 2019 The Tekton Authors.` to `// Copyright © 2020 The Tekton Authors.`